src/app/components/arlas-wui-root/arlas-wui-root.component.ts

```diff
@@ -286,14 +284,8 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
           this.mapglComponent.map.resize();
           this.resizeCollectionCounts();
           this.adjustVisibleShortcuts();
-          this.adjustCoordinates();
-        });
-      /** trigger 'resize' event after toggling sidenav  */
-      this.sidenavService.sideNavState
-        .pipe(takeUntil(this._onDestroy$))
-        .subscribe(res => {
-          this.onSideNavChange = res;
-          window.dispatchEvent(new Event('resize'));
+          this.adjustComponentsSize();
+          this.updateVisibleItems();
```

Comment:
`adjustComponentsSize()` contains a call to `updateVisibleItems`
I think you can remove `this.updateVisibleItems()` here